mobile/app/src/main/res/values/strings.xml

```diff
@@ -262,4 +262,12 @@
     </string>
     <string name="iitc_ce_mobile" translatable="false">IITC-CE Mobile</string>
     <string name="continue_in_english" translatable="false">Continue in English</string>
+    <string name="menu_delete_plugin">Delete</string>
+    <string name="delete_plugin_question">Delete plugin \"%s\"?</string>
+    <string name="plugin_deleted">Plugin \"%s\" deleted</string>
+    <string name="button_confirm_description">Yes</string>
+    <string name="button_cancel_description">No</string>
+    <string name="menu_share_plugin_download_url">Share download URL</string>
```

Comment:
is there a situation where we would want to share the update URL while having the download URL ?

Reply:
when it defers, the update URL is usually a meta.js file that's not that useful to share.

Reply:
Well... it's there for people to copy/share. Maybe it will be helpful for someone having problems with update because the URL differ and because the update URL is wrong.

Better to have this and not need it rather than need it and not have it.